website/content/docs/agent-and-proxy/agent/process-supervisor.mdx

```diff
@@ -6,7 +6,7 @@ description: >-
   into a process via environment variables using Consul Template markup.
 ---
 
-# Vault agent's process supervisor mode
+# Vault Agent's process supervisor mode
```

Comment:
Is it all init caps for "Process Supervisor"? eg "Vault Agent's Process Supervisor mode" as it is below

Reply:
Good catch! I'll update this